components/Storyblok/ComponentNotFound.js

```diff
@@ -0,0 +1,5 @@
+const ComponentNotFound = ({ component: { blok } }) => (
```

Comment:
New, very red and in your face, missing component component.

Reply:
Just what we need 😹